tests/test_extensions.py

```diff
@@ -26,5 +27,39 @@ class XViewset(mixins.ListModelMixin, viewsets.GenericViewSet):
 
     schema = generate_schema('x', XViewset)
     validate_schema(schema)
-    schema['components']['schemas']['X']['properties']['hash']['type'] == 'string'
-    schema['components']['schemas']['X']['properties']['hash']['format'] == 'byte'
+    assert schema['components']['schemas']['X']['properties']['hash']['type'] == 'string'
+    assert schema['components']['schemas']['X']['properties']['hash']['format'] == 'byte'
+
+
+def test_serializer_field_extension_geo(no_warnings):
```

Comment:
please move contib tests to `tests/contib/PACKAGE_NAME`

also would like to see a full schema test for that. have a look at test_oauth_toolkit for reference. makes it easier to inspect that the generated schema look reasonable.